.github/workflows/build-images.yaml

```diff
@@ -32,12 +32,26 @@ jobs:
           echo "IMG_TAGS=${GITHUB_REF_NAME/\//-} ${{ env.IMG_TAGS }}" >> $GITHUB_ENV
       - name: Set Authorino version
         id: authorino-version
+        run: |
+          echo "version=latest" >> $GITHUB_ENV
+      - name: Set Authorino git sha
+        id: authorino-git-sha
         run: |
           if [[ ${GITHUB_REF_NAME/\//-} =~ ^v[0-9]+\.[0-9]+\.[0-9]+(-.+)?$ ]]; then
-          echo "VERSION=${GITHUB_REF_NAME/\//-}" >> $GITHUB_ENV
+          echo "version=${GITHUB_REF_NAME/\//-}" >> $GITHUB_ENV
           else
```

Comment:
It looks like we're setting the version to `latest` always; then, in some cases, setting it again to the value of `$GITHUB_REF_NAME`. I wonder if `echo ... >> $GITHUB_ENV` replaces the existing entry with a new one or (as I would expect) adds another. Which entry will `${{ env.version }}` read? Probably what we want is to set `version=latest` only when it's not a semantic version and to `$GITHUB_REF_NAME` otherwise?

Reply:
You're right - we should add more logic so that `version` isn't in the github environment more than once at any given stage.